Jdmpview from Java 8.0.6.26 does not work with system core file from Java 8.0.5.37

$ <path>/sdks/pap6480sr6fp26-20210216_01-sdk/bin/jdmpview -core core.20210401.233037.3628.0001.dmp

> context
Available contexts (* = currently selected context) :

Source : file:///.../core.20210401.233037.3628.0001.dmp.zip_unpack/core.20210401.233037.3628.0001.dmp
        *0 : PID: 3628 :
                JRE 1.8.0 Windows Server 2012 R2 amd64-64


> !shrc name
!j9sharedclassconfig 0x000000000E5974B0

Error executing DDR command !shrc name : null
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
        at java.lang.reflect.Method.invoke(Method.java:508)
        at com.ibm.jvm.dtfjview.CombinedContext.executeDDRInteractiveCommand(CombinedContext.java:258)
        at com.ibm.jvm.dtfjview.CombinedContext.execute(CombinedContext.java:169)
        at com.ibm.jvm.dtfjview.Session.execute(Session.java:811)
        at com.ibm.jvm.dtfjview.Session.execute(Session.java:765)
        at com.ibm.jvm.dtfjview.tools.ToolsRegistry.executeJdmpviewCommand(ToolsRegistry.java:183)
        at com.ibm.jvm.dtfjview.tools.ToolsRegistry.execute(ToolsRegistry.java:107)
        at com.ibm.jvm.dtfjview.tools.ToolsRegistry.execute(ToolsRegistry.java:88)
        at com.ibm.jvm.dtfjview.tools.ToolsRegistry.recordAndExecute(ToolsRegistry.java:72)
        at com.ibm.jvm.dtfjview.Session.runInteractive(Session.java:739)
        at com.ibm.jvm.dtfjview.Session.run(Session.java:663)
        at com.ibm.jvm.dtfjview.DTFJView.launch(DTFJView.java:52)
        at com.ibm.jvm.dtfjview.DTFJView.main(DTFJView.java:46)
Caused by: java.lang.NoSuchFieldError: com/ibm/j9ddr/vm29/structure/SH_CacheMap.__ccTailOffset_
        at com.ibm.j9ddr.vm29.pointer.generated.SH_CacheMapPointer._ccTail(SH_CacheMapPointer.java:321)
        at com.ibm.j9ddr.vm29.tools.ddrinteractive.commands.ShrCCommand.dbgShrcReadConfig(ShrCCommand.java:1209)
        at com.ibm.j9ddr.vm29.tools.ddrinteractive.commands.ShrCCommand.getSharedCacheMetadataStart(ShrCCommand.java:1311)
        at com.ibm.j9ddr.vm29.tools.ddrinteractive.commands.ShrCCommand.run(ShrCCommand.java:105)
        at com.ibm.j9ddr.tools.ddrinteractive.Context.tryCommand(Context.java:229)
        at com.ibm.j9ddr.tools.ddrinteractive.Context.execute(Context.java:202)
        at com.ibm.j9ddr.tools.ddrinteractive.DDRInteractive.execute(DDRInteractive.java:356)
        at com.ibm.j9ddr.command.CommandReader.processLine(CommandReader.java:79)
        at com.ibm.j9ddr.tools.ddrinteractive.DDRInteractive.processLine(DDRInteractive.java:331)
        ... 16 more

--- a/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShrCCommand.java
+++ b/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShrCCommand.java
@@ -1202,9 +1202,14 @@ public class ShrCCommand extends Command
 					/*  _ccHead -------------> ccNext ---------> ccNext --------> ........---------> ccTail
 					 * (top layer)         (middle layer)     (middle layer)      ........         (layer 0)
 					 */
-					SH_CompositeCacheImplPointer compositeCacheImpl = cacheMap._ccTail();
-					for (int tmplayer = 0; tmplayer < layer; tmplayer++) {
-						compositeCacheImpl = compositeCacheImpl._previous();
+					SH_CompositeCacheImplPointer compositeCacheImpl;
+					try {
+						compositeCacheImpl = cacheMap._ccTail();
+						for (int tmplayer = 0; tmplayer < layer; tmplayer++) {
+							compositeCacheImpl = compositeCacheImpl._previous();
+						}
+					} catch (NoSuchFieldError | NoSuchMethodError e) {
+						compositeCacheImpl = cacheMap._cc();
 					}
 					if (compositeCacheImpl.notNull()) {
 						cacheStartAddress[layer] = compositeCacheImpl._theca();

